docker-compose.yml

```diff
@@ -69,6 +95,34 @@ services:
       - '$HOME/tweets-analyzer/es-integration/logs:/clients/es-integration/logs'
     networks:
       - spring-cloud-services-network
+  user-service:
+    container_name: user-service
+    depends_on:
+      - postgres
+      - config-server
+      - service-discovery
+    environment:
+      - EUREKA_SERVER=http://service-discovery:8761/eureka
+      - CONFIG_SERVER=http://config-server:8888
+    build:
+      context: ./clients/user-services/spring-data-jpa-user-service
+    ports:
+      - '8083:8083'
+    networks:
+      - spring-cloud-services-network
+  api-gateway:
+    container_name: api-gateway
+    depends_on:
+      - service-discovery
+    environment:
+      - EUREKA_SERVER=http://service-discovery:8761/eureka
+    build:
+      context: ./api-gateway
+    ports:
+      - '8762:8762'
```

Comment:
what sense of your api-gateway if you're producing all ports from each server to host machine?
the idea was to hide all services except api-gateway inside the docker process and provide only one port to end server.